### PR TITLE
Redux toolkit

### PR DESCRIPTION
**Pull request by Dachrono**
# 🎉 🎈 In this *Milestone* 🙋‍♂️ 🚩 ‼ :

- Follow the best GitHub flow. ✅
- Install linters. ✅
- No linters error. ✅
- Install redux toolkit. ✅ 
- Create redux logic folders. ✅
- Create store. ✅
- Create slices with initial state empty for books and categories. ✅
- Create reducer to add and remove books. ✅
- Create reducer to check under construction. ✅

### CODE REVIEW

File: src/redux/categories/categoriesSlice.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStaus: (state) => {
+      state.categories = 'under construction';
+    },
+  },
+});
+
```
Comment (rustam-tolipov) [inline src/redux/categories/categoriesSlice.js:16]: **You did an awesome job!**

- [ ] One more step to get approved. So I can see you exported categoriesSlice.reduce. Also, we need to export `checkStatus` as an action. Like this: 

```javascript
export const { addBook, removeBook } = booksSlice.actions;
``` 
  Kindly export `checkStatus` as an action and be ready to get approved.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @Dachrono,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Well-implemented code_
_Great readme_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/categories/categoriesSlice.js (hunk 1 above)
Comment (Dachrono) [reply src/redux/categories/categoriesSlice.js:16]: Solved

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Dachrono 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**